dictionaries/software-terms/src/software-terms.txt

```diff
@@ -817,6 +817,7 @@ imurmurhash
 incrementable
 incrementally
 incrementor
+indentless
```

Comment:
This isn't really a software term. Other than YamlFmt, I'm not finding it anywhere. I'll add it to `src/coding-terms.txt` as `indentless_array`.

Reply:
Thanks!